R/stats-factanal-tidiers.R

```diff
@@ -75,25 +106,35 @@ augment.factanal <- function(x, data, ...) {
   }
 
   # Place relevant values into a tidy data frame
-  tidy_df <- data.frame(.rownames = rownames(scores), data.frame(scores)) %>%
-    as_tibble()
+  if (has_rownames(scores)) {
+    tidy_df <- data.frame(.rownames = rownames(scores), data.frame(scores)) %>%
```

Comment:
I looked into this and I would prefer to not use it right now since I am actually not sure how that function is working. I will raise a separate issue maybe about its behavior.